doc/iterations/2025.6/diagram_read_only_visual_cue.adoc

```diff
@@ -0,0 +1,18 @@
+= Add the ability to give users an obvious visual cue as to whether the diagram is editable or read-only
+
+== Problem
+
+There is a lack of visual cues as to whether a diagram is in read-only or is editable.
+The grid background conditionned to the "snap to grid" state would be a good visual cue but it is hardcoded to be disabled by default.
```

Comment:
I really don't see why being read-only or not has anything to do with the state of snap to grid. Updating Sirius Web to better indicate that a diagram is read only, I can understand. Using snap to grid for that, I don't.